main/test/ca/uwaterloo/flix/library/TestGraph.flix

```diff
@@ -0,0 +1,34 @@
+namespace TestGraph {
+
+    /////////////////////////////////////////////////////////////////////////////
+    // closure                                                                 //
+    /////////////////////////////////////////////////////////////////////////////
+
+    @test
+    def closure01(): Bool & Impure =
```

Comment:
I was wondering if there would be a way to visualize the graphs?

Reply:
Jonathan did this: https://github.com/flix/flix/blob/master/examples/datalog/connect-graph.flix#L74 but maybe that is too much work.

Reply:
I could try doing something like that :-)
Then I can always stop, if it takes too long

Reply:
I stole some of Jonathans work and made arrows for directions. Is that what you had in mind?